app/src/main/java/com/humberto/tasky/auth/presentation/login/LoginScreen.kt

```diff
@@ -89,17 +92,23 @@ private fun LoginScreen(
     state: LoginState,
     onAction: (LoginAction) -> Unit
 ) {
-    RoundedBordersBackground(
+    val (
+        emailFocusRequester,
+        passwordFocusRequester,
+    ) = FocusRequester.createRefs()
+
+    val isEmailFocused = remember { mutableStateOf(false) }
+    val isPasswordFocused = remember { mutableStateOf(false) }
```

Comment:
Could it make sense to use the delegate here instead to avoid having to call `.value` all the time?
```
var myComposeState by remember { mutableStateOf("Hello world") }
// ...
myComposeState = "Bye bye world" // Updating the state
```